python/src/com/google/idea/blaze/python/issueparser/PyIssueParserProvider.java

```diff
@@ -130,7 +130,8 @@ private static int rankResult(PsiFile file, GlobalSearchScope projectScope, Sdk
       if (projectScope.contains(vf)) {
         return 0;
       }
-      return PythonSdkType.isStdLib(vf, sdk) ? 2 : 1;
+      return 2;
+      // return PythonSdkType.isStdLib(vf, sdk) ? 2 : 1;
```

Comment:
@mprobst Looks like the `isStdLib` function has been removed in the latest Python plugin. Any thoughts on how to handle this? The new version doesn't seem to have any relevant alternatives.